completions/ip

```diff
@@ -15,6 +15,19 @@ _comp_cmd_ip__netns()
     )"
 }
 
+_comp_cmd_ip__link_types()
+{
+    _comp_compgen_split -- "$(
+        {
+            ${1-ip} -c=never link help || ${1-ip} link help
+        } 2>&1 | command sed -e \
+            '/TYPE := /,/}/!d' -e \
+            's/.*{//' -e \
+            's/}.*//' -e \
+            's/|//g'
```

Comment:
```suggestion
            's/|/ /g'
```

Currently it seems to be separated by `<space>|<space>`, but it's safer to assume `|`. Or you could do `s/ | / /g`.

Reply:
Done